guides/change-management.md

```diff
@@ -0,0 +1,31 @@
+# Change management
```

Comment:
Noting in writing my concern about the name "change management" that I mentioned in our last call.

Reply:
Updated to Browser Support Exceptions Guide in https://github.com/cfpb/development/pull/165/commits/ecb097c4ce1d64a4bbcedcd537d2f783932cb11d